kinto/__main__.py

```diff
@@ -65,6 +67,16 @@ def main(args=None):
 
         init(config_file, backend)
 
+        # Install postgresql libraries if necessary
+        if backend == "postgresql":
+            try:
+                import psycopg2  # NOQA
+            except ImportError:
+                ret = call('pip install cliquet[postgresql]'.split())
```

Comment:
Note that if the venv is not activated it will fail. Cannot we call pip using a python command?